homeassistant/components/shelly/const.py

```diff
@@ -296,6 +296,10 @@ class BLEScannerMode(StrEnum):
     "flow_rate": SensorDeviceClass.VOLUME_FLOW_RATE,
     "water_pressure": SensorDeviceClass.PRESSURE,
     "water_temperature": SensorDeviceClass.TEMPERATURE,
+    "work_state": SensorDeviceClass.ENUM,
```

Comment:
If we add a mapping to `SensorDeviceClass.ENUM` here, we will also need a mapping of the `role` to `options` (possible states) and `translation_key`.